smoke-tests/images/servlet/build.gradle.kts

```diff
@@ -34,8 +34,6 @@ val targets = mapOf(
   "liberty" to listOf(
     ImageTarget(listOf("20.0.0.12"), listOf("hotspot", "openj9"), listOf("8", "11"), mapOf("release" to "2020-11-11_0736")),
     // running configure.sh is failing while building the image with Java 19
```

Comment:
@fmhwong do you know if this is still an issue, or can we (should we?) test with Java 19?

Reply:
Java 19 should be supported starting with version 22.0.0.10.